assets/airports/lrop.json

```diff
@@ -0,0 +1,715 @@
+                "26R": [["RARIT", "A25+|S210-"], "_FAF26R"],
+                "26L": [["LEVTA", "A25+|S210-"], "_FAF26L"]
+            },
+            "draw": [
+                ["TOSVI", "OP269", "OP268", "ESUXU", "ITVUK", "IVDEK", "OP262", "OP261", "LEVTA"],
+                ["TOSVI", "OP269", "OP268", "ESUXU", "ITVUK", "IVDEK", "OP262", "OP261", "RARIT"]
+            ]
+        },
+        "SORDU" : {
+            "icao": "SORDU",
+            "name": "Sordu Arrival",
+            "suffix": {"08R": "1U", "08L": "1U", "26R": "1X", "26L": "1X"},
+            "entryPoints": {
+                "SORDU": ["SOMOV" ]
+            },
+            "body": [["SORDU", "A100+"]],
+            "rwy": {
+                "08R": ["TUTIX", ["TEVRO", "A70+|S240-"], ["OP083", "A35-|S220-"], "OP082", "OP081", ["OBELA", "A25+|S210-"]],
+                "08L": ["TUTIX", ["TEVRO", "A70+|S240-"], ["OP083", "A35-|S220-"], "OP082", "OP081", ["DILAS", "A25+|S210-"]],
+                "26R": ["DONOX", "UTIDI", ["TUMOP", "A70+"], ["ESUXU", "S230-"], "ITVUK", ["IVDEK", "S220-"], "OP262", "OP261", ["RARIT", "A25+|S210-"]],
+                "26L": ["DONOX", "UTIDI", ["TUMOP", "A70+"], ["ESUXU", "S230-"], "ITVUK", ["IVDEK", "S220-"], "OP262", "OP261", ["LEVTA", "A25+|S210-"]]
+            },
+            "draw": [
+                ["SORDU", "TUTIX", "TEVRO", "OP083", "OP082", "OP081", "DILAS"],
+                ["SORDU", "TUTIX", "TEVRO", "OP083", "OP082", "OP081", "OBELA"],
+                ["SORDU", "DONOX", "UTIDI", "TUMOP", "ESUXU", "ITVUK", "IVDEK", "OP262", "OP261", "RARIT"],
+                ["SORDU", "DONOX", "UTIDI", "TUMOP", "ESUXU", "ITVUK", "IVDEK", "OP262", "OP261", "LEVTA"]
+            ]
+        },
+        "OBUGA" : {
+            "icao": "OBUGA",
+            "name": "Obuga Arrival",
+            "suffix": {"08R": "1U", "08L": "1U", "26R": "1X", "26L": "1X"},
+            "entryPoints": {
+                "OBUGA": ["KOMAN"]
+            },
+            "body": [["OBUGA", "A110+"]],
+            "rwy": {
+                "08R": [["TEVRO", "A70+|S240-"], ["OP083", "A35-|S220-"], "OP082", "OP081", ["OBELA", "A25+|S210-"]],
+                "08L": [["TEVRO", "A70+|S240-"], ["OP083", "A35-|S220-"], "OP082", "OP081", ["DILAS", "A25+|S210-"]],
+                "26R": ["UTIDI", ["TUMOP", "A70+"], ["ESUXU", "S230-"], "ITVUK", ["IVDEK", "S220-"], "OP262", "OP261", ["RARIT", "A25+|S210-"]],
+                "26L": ["UTIDI", ["TUMOP", "A70+"], ["ESUXU", "S230-"], "ITVUK", ["IVDEK", "S220-"], "OP262", "OP261", ["LEVTA", "A25+|S210-"]]
+            },
+            "draw": [
+                ["OBUGA", "TEVRO", "OP083", "OP082", "OP081", "DILAS"],
+                ["OBUGA", "TEVRO", "OP083", "OP082", "OP081", "OBELA"],
+                ["OBUGA", "UTIDI", "TUMOP", "ESUXU", "ITVUK", "IVDEK", "OP262", "OP261", "RARIT"],
+                ["OBUGA", "UTIDI", "TUMOP", "ESUXU", "ITVUK", "IVDEK", "OP262", "OP261", "LEVTA"]
+            ]
+        }
+    },
+    "spawnPatterns": [
```

Comment:
in-game acph: 69
real life acph: 12.4 (108200/365)/24 = 12.4 (3 signficant figures)
acph should be between 12.4 and 37.1

Please reduce the acph in-game